register-istanbul.js

```diff
@@ -3,14 +3,21 @@
  * To use, run mocha --require coffee-coverage/register-istanbul
  */
 var coffeeCoverage = require('./');
+var { resolve } = require('path');
 var coverageVar = coffeeCoverage.findIstanbulVariable();
 var writeOnExit = coverageVar == null ? true : null;
 
+var outFile = writeOnExit ? ((_ref = process.env.COFFEECOV_OUT) != null ? _ref : 'coverage/coverage-coffee.json') : null
+if (process.env.NYC_CONFIG) {
+    var config = JSON.parse(process.env.NYC_CONFIG);
+    outFile = resolve(config.cwd, config.tempDirectory, process.env.NYC_ROOT_ID + '.json');
+}
```

Comment:
This makes `nyc` aware of the report produced by `coffee-coverage`.
It generates the outFile in the proper place so `nyc` can parse it during its report phase.